person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..10_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    true if of_age(@age) == true || parent_permission == true
```

Comment:
- [ ] Kindly check on your of_age method is doesn't take any argument but in you can use services you have passed the age parameter to it.

- [ ] [optional] @Ol-create great work but you don't need to check if of age is equal to true since by default it will check it the same with parent permission, this will make your short and more readable
```suggestion
     of_age() || @parent_permission 
```